p2p_wallet/RxSwift_Deprecated/PendingTransaction.swift

```diff
@@ -91,6 +91,10 @@ struct PendingTransaction {
     let rawTransaction: RawTransactionType
     var status: TransactionStatus
     var slot: UInt64 = 0
+    
+    var isConfirmedOrError: Bool {
```

Comment:
Naming is a bit confusing for me. Is it "isFinished" or it is not the same?

Reply:
Yes. It is not the same. Transaction can be `sending` (from client to blockchain), `isConfirmed` (by 1 or some nodes), `isFinalized` (confirmed by all nodes) or `error`. In this context, we only check if it is confirmed (by some node or all node) or error. When a transaction `isConfirmedOrError`, it doesn't necessary mean `isFinalized` (confirmed by all nodes) but it can be confirmed by just some nodes or throw an error

Reply:
maybe "completed"? It doesn't mean it had been completed successfully